src/barrier-picker/SpreadBarrierCard.js

```diff
@@ -42,7 +42,7 @@ export default class SpreadBarrierCard extends Component {
                 <InputGroup
                     type="number"
                     label={`Amount per point (${currency})`}
-                    value={spreadInfo.amountPerPoint}
+                    defaultValue={spreadInfo.amountPerPoint}
```

Comment:
Have a look on other barrier?

Specifically the onStopLossChange and onStopProfitChange are actions that suppose to change the value of Input